### PR TITLE
Deprecate Gnarails 

### LINKED ISSUE #171: Deprecate Gnarails
> Isn't it rich?
> Are we a pair?
> Me here at last on the ground,
> You in mid-air...
> ...Send in... the clowns...

I believe we should deprecate `Gnarails`. As of #170, this gem is a thin wrapper over a bash script - the actual config files live in the `.gnarrc` repo, and this gem now simply pulls down the resulting `.rc` file and calls `rails new` with a reference to it. 

In `gnar-cli`, we have a [PR](https://github.com/TheGnarCo/gnar-cli/pull/31) that functionally makes that command line interface a superset of `gnarails` - if a little more verbose. 

Deprecating this would mean a few different things: 
 - updating the readme to point folks to Gnar-cli 
 - releasing a new and final version of the gem with a post-install deprecation message
 
 We should wait until the new version of `gnar-cli` is released before we do this.

### PR DESCRIPTION
**Pull request by alxjrvs**
```
The palm at the end of the mind,
Beyond the last thought, rises
In the bronze decor,

A gold-feathered bird
Sings in the palm, without human meaning,
Without human feeling, a foreign song.

You know then that it is not the reason
That makes us happy or unhappy.
The bird sings. Its feathers shine.

The palm stands on the edge of space.
The wind moves slowly in the branches.
The bird's fire-fangled feathers dangle down.

"Of Mere Being"
Wallace Stevens
```

This PR Deprecates `Gnarails` in favor of [`Gnar-CLI`](https://github.com/TheGnarCo/gnar-cli). 

Closes #171.

It bumps the version, adds a deprecation note to the Readme, and adds a post-install message that sends people to the newly consolidated tool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# This Gem has been Deprecated
+
+If you are looking to build rails apps with Gnarly Opinions, check out the `init` command of [`gnar-cli`].
```
Comment (mikestone14) [inline README.md:3]: should `gnar-cli` link to the Gnar CLI repo or somewhere similar?

Comment (alxjrvs) [reply README.md:3]: Duh, Yep!

Comment (alxjrvs) [inline README.md:3]:
```suggestion
If you are looking to build rails apps with Gnarly Opinions, check out the `init` command of [`gnar-cli`]('github.com/theGnarCo/gnar-cli').
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# This Gem has been Deprecated
+
+If you are looking to build rails apps with Gnarly Opinions, check out the `init` command of [`gnar-cli`]('github.com/theGnarCo/gnar-cli').
```
Comment (alxjrvs) [inline README.md:3]:
```suggestion
If you are looking to build rails apps with Gnarly Opinions, check out the `init` command of [`gnar-cli`](github.com/theGnarCo/gnar-cli).
```

File: gnarails.gemspec [hunk 3]
```diff
@@ -31,4 +31,14 @@ Gem::Specification.new do |spec|
...
   spec.metadata["rubygems_mfa_required"] = "true"
+  spec.post_install_message = %(
+    ==============================
+    THIS GEM HAS BEEN DEPRECATED!
+    ==============================
+    If you are looking to initialize
+    Rails apps with Gnarly Opinions,
+    check out Gnar-CLI:
+
+    github.com/theGnarCo/gnar-cli
```
Comment (alxjrvs) [inline gnarails.gemspec:42]:
```suggestion
    https://www.github.com/theGnarCo/gnar-cli
```

**Review by mikestone14 — APPROVED** [review]

**Pull request merged**